crates/compiler/gen_dev/src/generic64/aarch64.rs

```diff
@@ -532,8 +533,14 @@ impl Assembler<AArch64GeneralReg, AArch64FloatReg> for AArch64Assembler {
     }
 
     #[inline(always)]
-    fn jmp_imm32(_buf: &mut Vec<'_, u8>, _offset: i32) -> usize {
-        todo!("jump instructions for AArch64");
+    fn jmp_imm32(buf: &mut Vec<'_, u8>, offset: i32) -> usize {
+        if (-(1 << 27)..(1 << 27)).contains(&offset) {
```

Comment:
Will one of these be off by a bit? I am trying to think if that is correct or not. Cause it is like -128 to 127 is the same number of bits. So I think it is something of that nature here, but I guess this would be equivalent to something like -127 to 127, which should still be fine.